MTM-Backend/src/routes/v1/item/item.service.ts

```diff
@@ -0,0 +1,105 @@
+import { db } from "../../../utils/db.server";
+import type { ItemInputNoID, ItemType } from "../../../types/item";
+
+/**
+ * Get a items by id
+ * @param category
+ * @param name
+ * @returns an array of items based upon the given category and name or null if not found
+ * will return all the items if category and name if null
+ */
+export const getItems = async (
```

Comment:
The service file should handle direct interaction with the db. All the business logic should be handled in the router. Again, please look at the mentioned PR for user and organizations for example.